include/JSystem/JGadget/std-list.hpp

```diff
@@ -2,56 +2,56 @@
 #define JGADGET_LIST_HPP
 
 #include <JSystem/JGadget/std-memory.hpp>
+#include <JSystem/JGadget/define.h>
 #include <types.h>
 
 namespace JGadget {
 
 template <class T, class A = TAllocator<T> > class TList {
 public:
 	struct TNode_ {
-		TNode_* mNext;
-		TNode_* mPrev;
+		TNode_* pNext_;
+		TNode_* pPrev_;
 	};
 
-	TNode_* CreateNode_(TNode_* next, TNode_* prev, const T& value)
+	TNode_* CreateNode_(TNode_* pNext, TNode_* pPrev, const T& value)
 	{
-		// Debug assert leftovers confirmed through TP. Don't touch.
-		(void)0;
-		(void)0;
+		// clang-format off
+        JGADGET_ASSERT(pNext!=0);
+        JGADGET_ASSERT(pPrev!=0);
```

Comment:
theses two lines must use tabs instead of spaces even though they are not formatted